nautobot/apps/__init__.py

```diff
@@ -1,11 +1,14 @@
 """Core app functionality."""
 
+from nautobot.core.apps import NautobotConfig
 from nautobot.core.signals import nautobot_database_ready
 from nautobot.core.settings_funcs import ConstanceConfigItem
 from nautobot.extras.plugins import NautobotAppConfig
 
+
 __all__ = (
     "ConstanceConfigItem",
     "nautobot_database_ready",
     "NautobotAppConfig",
+    "NautobotConfig",
```

Comment:
I don't think `NautobotConfig` should be included - apps should always be using its subclass `NautobotAppConfig` that's already here.